data/human/human missions.txt

```diff
@@ -6891,3 +6891,335 @@ mission "Returning to Paradise"
+			choice
+				`	"I think you're right."`
+				`	"What about doing both?"`
+					goto plan
+
+			`	"And you know they'll use those poor people as human shields. Say we-" A chirp from the proximity alert cuts him off.`
+				goto showtime
+
+			label plan
+			`	"Now you're talking! But it would take a lot of planning to do it without killing the people we are trying to liberate. You know they'll use those poor people as human shields. Say we-" A chirp from the proximity alert cuts him off.`
+
+			label showtime
+			`	"Showtime!" he exclaims, rubbing his hands together. Both of you watch as a capital ship accompanied by a small flotilla of Quicksilver escorts, all in pirate colors, land near the security detail, probably deafening them with the sound of their repulsors. As soon as the loading ramp is down, the security detail gets to work herding the slaves from one ship to another; hitting, shoving, and shouting at anyone that moves too slowly for their liking.`
+			`	"Pigs," blurts out Bunker. He zooms in on two men talking a few meters away from everyone else. One is in a suit, the other a black hooded robe. "Must be the leaders," he whispers as if they can hear him. Just then, the hooded man looks up to the sky as if he sees something. He motions, and the Quicksilvers fire up their engines.`
+			`	"Oh, I don't like the look of this," says Bunker.`
+			choice
+				`	"Keep filming."`
+				`	"Call back the drones immediately."`
+					goto "call drones"
+				`	"I think we'll have to leave the drones behind."`
+					goto "no can do"
+				`	"Did he see us?"`
+					goto "see us"
+
+			`	"I don't know. I think we oughta' get going," says Bunker, grimacing.`
+			choice
+				`	"That's an order!"`
+				`	"Okay. Call the drones back."`
+					goto "call drones"
+
+			`	"You're the captain," Bunker responds in a sore tone. Meanwhile, you continue to watch the display. It takes a couple minutes before the last of the slaves are loaded, followed by the security team. The two men in the suit and hood are the last to board their ships.`
+			`	You barely notice the humming of thrusters from outside just as the <ship> receives a hail of particle cannon fire from a pirate Quicksilver, appearing seemingly from nowhere as they whittle the shields down in a split second. You shunt the <ship>'s repulsors to full blast to get into orbit, but another pair of Quicksilvers send another volley of particle fire that breaks through and takes the repulsors offline. The last thing you experience is the ship slowing to a halt, pausing in the air for just a moment before the pilot seat goes out underneath you, the cockpit filled with screams as the <ship> makes a one way trip towards the surface.`
+				die
```

Comment:
I'm not so sure this should result in insta-death. The thought that a couple of Quicksilvers armed with particle cannons could tear through a capital ship's shield before you have time to react or get away... it beggars belief. I think the player should be launched into orbit and have to fight for their life.

If you really, _really_ want to force insta-death, then you should change the text to remove references of going through the ship's shields. The player must have had their shields down to avoid interfering with the drone signals, or the ship was intentionally on low power mode to avoid attracting attention. Ideally, in fact, I would have the player in a vulnerable situation. Perhaps they stuck their head out of the cockpit or something like that. Otherwise, you're going to end up in an unrealistic situation where someone is scoffing at the fact that three Quicksilvers took down their Hurricane, for example.

Reply:
I will think about rewriting it.

Reply:
Ok, what I could do is let the player run away, but decline this mission (`flee`). This allows you to complete this mission chain, but you won't be able to continue doing Adelita missions down the line since you weren't able to recover the footage. Separate missions will take care of the different dialogue that will arise due to this, and you won't recieve as much money as before. After all, the player can't always be forced to pick the right option, or otherwise we're dictating to them what we want the story to go in a supposedly multi-choice game. I could even let the player re-establish themselves with Adelita in a follow-up mission, though I'd rather get the rest of #5112 done first.